plugins/inputs/http/sample.conf.in

```diff
@@ -1,6 +1,14 @@
 # Read formatted metrics from one or more HTTP endpoints
 [[inputs.http]]
-  ## One or more URLs from which to read formatted metrics
+  ## One or more URLs from which to read formatted metrics.
+  ##
+  ## HTTP requests over Unix domain sockets can be specified via the "http+unix" or "https+unix" schemes.
+  ## Request URLs should have the following form:
+  ##
+  ##    http+unix:///path/to/service.sock:/api/endpoint
+  ## Example:
+  #     http+unix:///run/user/420/podman/podman.sock:/d/v4.0.0/libpod/pods/json
+  ## Note: The path to the Unix domain socket and the request endpoint are separated by a colon (":").
```

Comment:
I would move this out of the config and into the README below the sample.conf include. It now reads like you _have to_ use a unix socket...

Reply:
@srebhan Having the documentation about the URLs in the sample.conf helps the user when they do something like below to generate a sample config:
 ```
 $ telegraf config create --section-filter "inputs" --input-filter "http"
 ```
They have the documentation about the supported URL schemes right there for them to refer.

I can move it out of the `sample.conf` but I think, what might be better is to just include examples with both the "normal" scheme and "http+unix" scheme.

```
urls = [
    "http://localhost/metrics",
    "http+unix:///var/run/docker.sock:/v1.42/containers/json"
]      
```
Please let me know if this works.

Reply:
I'm fine with adding these as examples to the config, but I would like to see the explanation in the README rather than in the `sample.conf`.

Reply:
Done!